src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should use the same configuration as the other fixtures, see https://github.com/test-editor/rest-fixture/blob/develop/src/test/resources/log4j2.xml